Source/DafnyDriver/DafnyDriver.cs

```diff
@@ -391,13 +391,13 @@ public enum CommandLineArgumentsResult {
         return exitValue;
       }
 
-      if (0 <= options.VerifySnapshots && lookForSnapshots) {
+      if (0 < options.VerifySnapshots && lookForSnapshots) {
```

Comment:
Is this a bug fix? And does it mean that we've accidentally been calling `ExecutionEngine.LookForSnapshots()` by default since the introduction of this code?

Reply:
Yes it's a bug fix but possibly only for a latent bug. 

According to the documentation, `0` means the future is turned off. `options.VerifySnapshots` is `-1` by default, which makes the previous code work well unless someone explicitly assigns `0` to this option.